### PR TITLE
Allow import of `isochrones` on readthedocs

### LINKED ISSUE #176: `import isochrones` fails on readthedocs
Love the package! I'm trying to document my own package which uses `isochrones` but there seems to be a bug that causes my build to fail.

When the environment variable "READTHEDOCS" is `True`, the package fails on import. This seems to be because you define `LOG_ONE_OVER_ROOT_2PI` using numpy but **you don't import it** when `on_rtd=True` (see [here](https://github.com/timothydmorton/isochrones/blob/master/isochrones/observation.py#L37))

To reproduce:
```
export READTHEDOCS=True
python
    import isochrones
```
And you'll get the error:
>> LOG_ONE_OVER_ROOT_2PI = np.log(1.0 / np.sqrt(2 * np.pi))
     NameError: name 'np' is not defined

### PR DESCRIPTION
**Pull request by TomWagg**
Super small PR! I've moved the definition of `LOG_ONE_OVER_ROOT_2PI` to prevent a crash during import whilst on readthedocs - this will fix #176.

(I also changed a "is" to "==" to prevent warnings in the latest python version)

### CODE REVIEW

**Comment by TomWagg** [comment]
Hi @timothydmorton, just wanted to check in on this PR - should be super quick but it'll let me build my docs for a package that uses `isochrones` 🙂

**Review by timothydmorton — APPROVED** [review]
Sure, looks great; go for it!

**Comment by TomWagg** [comment]
Thanks @timothydmorton! - but I don't have write access so I can't merge the PR 😅

**Pull request merged**
